### PR TITLE
Reformatting Kinesis IO to comply with official code style

### PR DESCRIPTION
**Pull request by pawel-kaczmarczyk**
Be sure to do all of the following to help us incorporate your contribution
quickly and easily:

 - [ ] Make sure the PR title is formatted like:
   `[BEAM-<Jira issue #>] Description of pull request`
 - [x] Make sure tests pass via `mvn clean verify`.
 - [ ] Replace `<Jira issue #>` in the title with the actual Jira issue
       number, if there is one.
 - [ ] If this contribution is large, please file an Apache
       [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

---

### CODE REVIEW

**Comment by jbonofre** [comment]
R: @jbonofre

**Comment by pawel-kaczmarczyk** [comment]
@jbonofre Can this one be merged already? It blocks other work in KinesisIO

**Comment by jbonofre** [comment]
I'm still reviewing it. Give me some more time please. Thanks !

**Comment by pawel-kaczmarczyk** [comment]
Sure, I just thought you have already reviewed it.

**Comment by pawel-kaczmarczyk** [comment]
Hi @jbonofre 
It's just a reminder about this MR ;)

**Comment by jbonofre** [comment]
Sorry ;) I'm resuming my review ;)

**Comment by jbonofre** [comment]
I'm resuming the review now. Thanks for your patience.

File: sdks/java/io/kinesis/src/main/java/org/apache/beam/sdk/io/kinesis/CheckpointGenerator.java [hunk 1]
```diff
@@ -17,14 +17,14 @@
...
 import java.io.Serializable;
 
 /**
  * Used to generate checkpoint object on demand.
  * How exactly the checkpoint is generated is up to implementing class.
  */
 interface CheckpointGenerator extends Serializable {
-    KinesisReaderCheckpoint generate(SimplifiedKinesisClient client)
-            throws TransientKinesisException;
+
```
Comment (jbonofre) [inline sdks/java/io/kinesis/src/main/java/org/apache/beam/sdk/io/kinesis/CheckpointGenerator.java]: Useless extra space (not a big deal anyway).

**Comment by pawel-kaczmarczyk** [comment]
Thanks @jbonofre!

**Pull request closed without merging**